### ISSUE TITLE
EKS Nodegroups documentation, not showing Inputs/Outputs

### ISSUE THREAD

**Issue by axblum** [pulumi/pulumi-aws#3799]
File: [themes/default/content/registry/packages/aws/api-docs/eks/nodegroup/_index.md](https://www.pulumi.com/registry/packages/aws/api-docs/eks/nodegroup/)

Outputs and Inputs are missing from this resource type.

**Comment by t0yv0** [comment]
Thank you @sean1588 ! We're looking at this briefly with @corymhall it appears that the information is present on the page but markup is broken so some of the HTML codes get rendered in the YAML pane. We're going to look if Description in schema.json is invalid markdown and see if there's any validations we can add. But it could also be an issue with the registry downstream of the repo. We'll take a look and work with you to fix it.

**Comment by sean1588** [comment]
>it appears that the information is present on the page but markup is broken so some of the HTML codes get rendered in the YAML pane.

thank you @t0yv0! you know what, we actually resolved this issue. I just did a build with the latest pu/pu to confirm and looks like this actually got fixed by one of the recent issues we addressed.  So the page actually looks great, I am just waiting on pu/pu to release so we can update the dependency in registry to deploy the more recent fixes.

<img width="821" alt="Screen Shot 2024-04-11 at 7 46 17 AM" src="https://github.com/pulumi/pulumi-aws/assets/16751381/31ad33e0-cccd-4ff3-847d-755ade8d90e8">

**Comment by sean1588** [comment]
yeah I'll transfer this issue back over to our side then, since this fix will get deployed once we are able to update the dependencies on our end.
